compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -355,6 +357,18 @@ function finishPrimaryExpr(Tokenizer tok, Expr expr, Position startPos) returns
     }
 }
 
+function parseIdentifierOrMethodName(Tokenizer tok) returns string|err:Syntax {
+    Token? t = tok.current();
+    if t is SpecialMethodName {
+        check tok.advance();
+        if tok.current() == "(" {
```

Comment:
Can't we use `check tok.expect("(");`

Reply:
If we do so we won't be able to check the token in line 346 since the tokenizer will advance right?